src/handbook/development/releases/planning.md

```diff
@@ -56,7 +55,7 @@ Issues are the building blocks of planning activities, helping the team to manag
 
   - **Feature Requests**: suggestions or ideas submitted by users or stakeholders for new functionalities, enhancements, or improvements to the existing software or system. Feature requests should be evaluated, prioritized, and potentially incorporated into the product roadmap, often being transformed into Epics or Stories for implementation in future releases.
 
-Whenever an issue is raised, it will be reviewed by the Product Manager/CTO and added
+Whenever an issue is raised, it will be reviewed by the Product Manager / Engineering Manager and added
```

Comment:
Just want to voice that whilst I'm okay with this, I still think it's Product Manager's role to do this scheduling, but EM in absence of PM is sensible.